cluster.go

```diff
@@ -1484,7 +1484,7 @@ func (c *Cluster) AddFile(reader *multipart.Reader, params *api.AddParams) (cid.
 	if params.Shard {
 		dags = sharding.New(c.rpcClient, params.PinOptions, nil)
 	} else {
-		dags = local.New(c.rpcClient, params.PinOptions)
+		dags = single.New(c.rpcClient, params.PinOptions, false)
```

Comment:
You have the `params.Local` at hand to pass it in rather than hardcoding to false.